third_party/move/tools/move-package/tests/test_sources/compilation/std-lib-override/Move.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "std-lib-conflicts"
+version = "1.0.0"
+authors = []
+
+[addresses]
+A = "0x42"
+
+[dev-addresses]
+
+[dependencies.AptosFramework]
+git = "https://github.com/aptos-labs/aptos-core.git"
+rev = "devnet"
+subdir = "aptos-move/framework/aptos-framework"
+
+[dependencies.B]
+local = "./deps_only"
```

Comment:
Nit: the directory name "deps_only" seems a bit confusing to me. Wonder if it would be better to rename this to "intermediate_dep".

Reply:
The test runner checks for directories containing "deps_only", and doesn't build the packages under that directory.